llvm/lib/Transforms/InstCombine/InstCombineCasts.cpp

```diff
@@ -436,6 +436,67 @@ static Instruction *foldVecTruncToExtElt(TruncInst &Trunc,
   return ExtractElementInst::Create(VecInput, IC.Builder.getInt32(Elt));
 }
 
+// Whenever an element is extracted from a vector, and then truncated,
```

Comment:
```suggestion
/// Whenever an element is extracted from a vector, and then truncated,
```
Should use `///` for func header comments.